aten/src/ATen/native/UpSample.h

```diff
@@ -447,7 +447,7 @@ static inline void compute_source_index_and_lambda(
     lambda0 = static_cast<scalar_t>(1);
     lambda1 = static_cast<scalar_t>(0);
   } else {
-    using accscalar_t = at::acc_type<scalar_t, false>;
+    using accscalar_t = at::acc_type<scalar_t, true>;
```

Comment:
nit: why don't we use `opmath_t` here that doesn't have weird discrepancy between use_cuda and not?